barman/cloud_providers/aws_s3.py

```diff
@@ -140,7 +140,7 @@ def _extra_upload_args(self):
         Return a dict containing ExtraArgs to be passed to certain boto3 calls
 
         Because some boto3 calls accept `ExtraArgs: {}` and others do not, we
-        return a nexted dict which can be expanded with `**` in the boto3 call.
+        return a nested dict which can be expanded with `**` in the boto3 call.
```

Comment:
I'm less confident about this (it was an automatically suggested item)

Reply:
I think nested is the right thing here :+1: